docs/tutorials/pytorch-mnist-tutorial.txt

```diff
@@ -16,12 +16,10 @@ Overview
 
 To use a PyTorch model in Determined, you need to port the model to Determined's API. For most models, this porting process is straightforward, and once the model has been ported, all of the features of Determined will then be available: for example, you can do :ref:`distributed training <multi-gpu-training>` or :ref:`hyperparameter search <hyperparameter-tuning>` without changing your model code, and Determined will store and visualize your model metrics automatically.
 
-When training a PyTorch model, Determined provides a built-in training loop that feeds batches of data into your forward pass, performs backpropagation, and computes training metrics. Determined also handles checkpointing, log management, and device initialization. To plug your model code into the Determined training loop, you define methods to perform the following tasks:
+When training a PyTorch model, Determined provides a built-in training loop that feeds batches of data into your batch training function, where you can perform forward pass, backpropagation, and training metrics computation. Determined also handles checkpointing, log management, and device initialization. To plug your model code into the Determined training loop, you define methods to perform the following tasks:
```

Comment:
... that feeds each batch of data into your `train_batch` function, where you will perform the forward pass, the backpropagation, and the computation of training metrics for that batch.